src/Client/LibevNetProvider.hpp

```diff
@@ -347,7 +347,7 @@ LibevNetProvider<BUFFER, Stream>::close(Stream_t &strm)
 	}
 }
 
-static void
+static inline void
```

Comment:
I don't think this is the right fix. You should rather make this function a private static member of the `LibevNetProvider` class.

Reply:
I don't see any problems here, but fixed.

Reply:
The problem is you shouldn't be leaking implementation details, when possible.